src/backend/api/meals.js

```diff
@@ -2,14 +2,199 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+
+router.get("/", async(request, response) => {
+
+    try {
+        // knex syntax for selecting things. Look up the documentation for knex for further info
+        const meals = await knex("meals");
+
+        if (request.query.hasOwnProperty('id')) {
+
+            const requestId = Number(request.query.id);
+            const requestedMeal = await knex('meals')
+                .select()
+                .where('id', '=', requestId);
+
+            console.log(requestedMeal);
+            console.log(typeof(requestedMeal));
```

Comment:
Good to do some console log while coding / debugging, not so good when submitting the HW! 😀 
🗑️